ouroboros-consensus/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -894,7 +976,7 @@ prop_Mempool_idx_consistency (Actions actions) =
     emptyTestSetup = TestSetup
       { testLedgerState = testInitLedger
       , testInitialTxs  = []
-      , testMempoolCap  = MempoolCapacity 1000
+      , testMempoolCap  = MempoolCapacity 1000000
```

Comment:
Perhaps it'd be a good idea to instead define `MempoolCapacity` as a sum type with a `NoMempoolCapacity` constructor that we can utilize in tests that don't really care about capacity.

Reply:
We can just use `maxBound` in the tests, if that's ever an issue in the tests, then there's something wrong with the tests (or the mock transaction sizes).

Reply:
Maybe it's good time to address #745 too?

Reply:
Yeah, for sure. I'll address #745 in a follow up PR.